doc/developers.rst

```diff
@@ -1166,6 +1166,39 @@ exchange before the loop) or add existing halo exchanges after a loop
 (as an increase in depth will only make it more likely that a halo
 exchange is no longer required after the loop).
 
+Kernel Transformations
+++++++++++++++++++++++
+
+When a kernel is transformed we need to ensure that the PSy layer is
```

Comment:
The first sentence is a bit of a throwaway i.e. it tells us something to worry about but does not say how this is managed. Either elaborate (how is this done and where is this done) or remove.

Reply:
Good spot. Removed.